source/agora/common/EnrollmentManager.d

```diff
@@ -205,6 +205,80 @@ public class EnrollmentManager
         this.db.execute("DELETE FROM validator_set WHERE key = ?", enroll_hash[]);
     }
 
+    /***************************************************************************
+
+        In validatorSet DB, return the enrolled block height.
+
+        Params:
+            enroll_hash = key for an enrollment block height
+
+        Returns:
+            the enrolled block height.
+            Return size_t if no matching key exists.
+
+    ***************************************************************************/
+
+    public size_t getEnrolledHeight (const ref Hash enroll_hash) @safe
+    {
+        return () @trusted {
```

Comment:
You can just mark the entire function as `@trusted`, it is the very same effect as making it `@safe` and having all of the code inside wrapped in a `@trusted` delegate.

Reply:
applied the suggestion.